MEMBER_REPRESENTATION.md

```diff
@@ -0,0 +1,9 @@
+# Member Representation
+
+The following document lists OpenJS Foundation members and their representation to any standards organizations:
+
+* [TC39]
+  * <firstname> <lastname> (#issue)
```

Comment:
`<firstname> <lastname>` is a poor fit for names from many cultures.

Maybe `<fullname>`

I believe that term is used in vCard to refer to a non-abbreviated name.